pulp_ansible/app/migrations/0050_crossrepositorycollectionversionindex.py

```diff
@@ -1,9 +1,205 @@
 # Generated by Django 3.2.18 on 2023-03-20 13:08
 
+from semantic_version import Version
+
+from django.db.models import Q
+
+from pulp_ansible.app.models import (
```

Comment:
models can't be imported. You need to get them from `apps` (example: https://github.com/pulp/pulp_ansible/blob/main/pulp_ansible/app/migrations/0035_deprecation_content.py#L94)